server/src/internalClusterTest/java/org/opensearch/search/profile/query/QueryProfilerIT.java

```diff
@@ -547,6 +602,9 @@ public void testPhrase() throws Exception {
                     assertNotNull(result.getLuceneDescription());
                     assertThat(result.getTime(), greaterThan(0L));
                     assertNotNull(result.getTimeBreakdown());
+                    if (concurrentSearchEnabled) {
+                        testQueryProfileConcurrentSearchStats(result);
+                    }
```

Comment:
1. We can rename `testQueryProfileConcurrentSearchStats` to `assertQueryProfileResult`. `test` pefix is usually used for a test method.
2. Let's move `concurrentSearchEnabled` check inside `assertQueryProfileResult` where for non-concurrent case as well, we can verify that min/max/avg slice time is null and size of breakdown map.

Reply:
Hey @sohami, thanks for the review. The matter has been resolved now.